pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -186,6 +190,11 @@ func isSPInList(name string, spList []StoragePoolInfo) bool {
 	return false
 }
 
+func stampPVCWithError(ctx context.Context, client kubernetes.Interface, curPVC *v1.PersistentVolumeClaim, errAnnotation string, err error) error {
+	_ = setPVCAnnotation(ctx, errAnnotation, client, curPVC)
```

Comment:
`setPVCAnnotation` returns error. Do you want to log that error and then return it?